recipes/smurff/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export CMAKE_ARGS="-DENABLE_MKL=ON -DCMAKE_INSTALL_PREFIX=$PREFIX -DENABLE_MPI=OFF"
```

Comment:
Extend the existing `CMAKE_ARGS`
```suggestion
export CMAKE_ARGS="${CMAKE_ARGS} -DENABLE_MKL=ON -DCMAKE_INSTALL_PREFIX=$PREFIX -DENABLE_MPI=OFF"
```